molgenis-data/src/main/java/org/molgenis/data/meta/model/AttributeMetadata.java

```diff
@@ -26,7 +26,7 @@
 	public static final String NAME = "name";
 	public static final String ENTITY = "entity";
 	public static final String SEQUENCE_NR = "sequenceNr";
-	public static final String TYPE = "type";
+	public static final String TYPE = "dataType";
```

Comment:
I'm quiet sure ```type``` is used in a lot of freemarker templates and JavaScript files. Did you check if there are references that require updating?